tox.ini

```
@@ -18,7 +18,7 @@ commands =
     codespell -I whitelist.txt src tests packages --skip 'tests/eest'
     ruff check src tests packages --exclude tests/eest
     ruff format src tests packages --check --exclude tests/eest
-    mypy src tests --exclude "tests/json_infra/fixtures/*|tests/eest/*" --namespace-packages  # TODO: add packages directory
+    mypy src tests packages --exclude "tests/json_infra/fixtures/*|tests/eest/*" --namespace-packages
```

Now that we have `files` in `pyproject.toml`, can we omit the paths here and just run:

```suggestion
    mypy
```

---

Yep that makes perfect sense, pushed 👍🏼